opsdroid/events.py

```diff
@@ -115,3 +123,13 @@ def __init__(self, text, user, room, connector, raw_message=None):
         if 'thinking-delay' in self.connector.configuration:
             await self._thinking_delay()
         return await self.connector.react(self, emoji)
+
+
+class Image(Message):
+    def __init__(self, image_url, user, room, connector):
+        self.created = datetime.now()
```

Comment:
This is a godo idea. We should move this up into the `Event` as it makes sense to have it in everything.

Reply:
Agreed. So far I've just made `Event` basically empty and I figure we'll move overlapping functionality into it as we find it. So please keep pointing it out 😃